docs/controllers/services/examples/http2-with-cert-nginx.yml

```diff
@@ -2,10 +2,11 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: https-with-cert-minimal
+  name: http2-with-cert-minimal
   annotations:
+    # HTTP2 port annotation for 443 needed since it defaults to HTTPS.
+    service.beta.kubernetes.io/do-loadbalancer-http2-ports: "443"
```

Comment:
Can you make this a separate example? I think it's still nice to have an example without using the `http2-ports` annotation for the simple case.

Reply:
Good point.

I changed the example by swapping the HTTP2 port annotation with a protocol annotation as we have the port case covered by the pass-through example already. LMK if that makes sense.

Reply:
WFM :+1: